avm/res/api-management/service/main.bicep

```
@@ -277,7 +281,7 @@ module service_apis 'api/main.bicep' = [
       authenticationSettings: api.?authenticationSettings
       format: api.?format ?? 'openapi'
       isCurrent: api.?isCurrent
-      protocols: api.?protocols
+      protocols: api.?protocols ?? ['https']
```

```suggestion
      protocols: api.?protocols
```
Anyways the default of the parameter in the child module. Same is true for the `format` property btw

---

Agree!